nextcord/flags.py

```diff
@@ -1249,3 +1249,60 @@ def active(self):
         .. versionadded:: 2.4
         """
         return 1 << 24
+
+
+@fill_with_flags()
+class MemberFlags(BaseFlags):
+    """Wraps up Discord's Member Flags.
+
+    .. container:: operations
+
+        .. describe:: x == y
+
+            Checks if two MemberFlags are equal.
+        .. describe:: x != y
+
+            Checks if two MemberFlags are not equal.
+        .. describe:: hash(x)
+
+            Return the flag's hash.
+        .. describe:: iter(x)
+
+            Returns an iterator of ``(name, value)`` pairs. This allows it
+            to be, for example, constructed as a dict or a list of pairs.
+            Note that aliases are not shown.
+
+    .. versionadded:: 2.4
+    """
+
+    @flag_value
+    def did_rejoin(self):
+        """:class:`bool`: Whether this member rejoined this guild.
+
+        .. versionadded:: 2.4
```

Comment:
Whenever we add a new class with methods, those initial methods don't have a `versionadded` in their docstrings because they were added in with their parent class.